sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.implementation;
+
+import java.util.Iterator;
+import java.util.Objects;
+
+/**
+ * Utility method class.
+ */
+public final class Utility {
+    private Utility() {
+    }
+
+    /**
+     * Verify that list of documents are not null or empty. Otherwise, throw exception.
+     *
+     * @param documents A list of documents.
+     */
+    public static void inputDocumentsValidation(Iterable<?> documents) {
+        Objects.requireNonNull(documents, "'documents' cannot be null.");
+        final Iterator<?> iterator = documents.iterator();
+        if (!iterator.hasNext()) {
+            throw new IllegalArgumentException("'documents' cannot be empty.");
```

Comment:
`throw logger.logExceptionAsWarning(...)`

Also, add a section for throws javadoc for the respective methods where this is called.